### PR TITLE
whatsnew

### PR DESCRIPTION
**Pull request by jburel**
update page for 5.5
cc @mtbc

### CODE REVIEW

File: omero/sysadmins/whatsnew.rst [hunk 1]
```diff
@@ -1,35 +1,23 @@
-What's new for OMERO 5.4 for sysadmins
+What's new for OMERO 5.5 for sysadmins
 ======================================
 
 - :doc:`version-requirements` has been updated to reflect other changes in
-  version support for 5.4.0 and tentative plans for 5.5.0.
+  version support for 5.5.0 and tentative plans for 5.6.0.
+
+- Walkthrough have been added for
```
Comment (mtbc) [inline omero/sysadmins/whatsnew.rst]:
```suggestion
- A walkthrough has been added for
```

File: omero/sysadmins/whatsnew.rst [hunk 2]
```diff
@@ -1,35 +1,23 @@
...
+- Add configuration option to fix OpenSSL issue.
 
-- The Public user is restricted to GET requests by default. This can be
-  changed by setting the new configuration property
-  :property:`omero.web.public.get_only`.
+- Improve the documentation of :doc:`in-place-import`.
 
-- It is now possible to include an html template in the base `core_html` 
-  template so it can be used for all webclient pages.
+- Add documentation on how to configure websockets see :doc:`websockets`.
```
Comment (mtbc) [inline omero/sysadmins/whatsnew.rst]:
```suggestion
- Add documentation on how to configure websockets, see :doc:`websockets`.
```

File: omero/sysadmins/whatsnew.rst [hunk 3]
```diff
@@ -1,35 +1,23 @@
...
 
-- The OMERO.web part of the :program:`omero admin diagnostics` command has
-  been moved to :program:`omero web diagnostics` as OMERO.web can be installed
-  separately from the OMERO.server.
+- Improve documentation on how to configure OMERO.web see :doc:`client-server-ssl`.
```
Comment (mtbc) [inline omero/sysadmins/whatsnew.rst]:
```suggestion
- Improve documentation on how to configure OMERO.web, see :doc:`client-server-ssl`.
```

File: omero/users/whatsnew.rst [hunk 4]
```diff
@@ -1,22 +1,12 @@
...
 
-- Improvements to OMERO.web including the display of fields within Wells, handling of float
-  images with small dynamic range and the ability to export images in plate as OME-TIFF as well as a number of minor bug fixes.
-
-- The 'Reverse Intensity' command in the image viewers has been renamed to
-  'Invert'.
-
-- The display of Tables data removed in 5.3 has been added back to the Image Viewer.
+- Improvements to OMERO.web including better support for public users and the introduction
+  of Advanced Search to allow and/or search options
```
Comment (mtbc) [inline omero/users/whatsnew.rst]:
```suggestion
  of Advanced Search to allow and/or search options.
```

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/62/console is green

File: omero/developers/whatsnew.rst [hunk 5]
```diff
@@ -1,53 +1,23 @@
...
+- https://github.com/ome/omero-blitz
+- https://github.com/ome/omero-gateway-java
+- https://github.com/ome/omero-blitz-plugin
+- https://github.com/ome/omero-insight
+- https://github.com/ome/omero-matlab
+- https://github.com/ome/omero-javapackager-plugin
+- https://github.com/ome/omero-api-plugin
+
+This has the goal of enabling more fine-grained releases.
+A new restriction is that the names of server configuration properties
```
Comment (mtbc) [inline omero/developers/whatsnew.rst:23]: Might need a linebreak? Best not ending the previous paragraph with this.

File: omero/sysadmins/whatsnew.rst [hunk 6]
```diff
@@ -1,35 +1,23 @@
-What's new for OMERO 5.4 for sysadmins
+What's new for OMERO 5.5 for sysadmins
 ======================================
 
 - :doc:`version-requirements` has been updated to reflect other changes in
```
Comment (mtbc) [inline omero/sysadmins/whatsnew.rst]:
```suggestion
- :doc:`version-requirements` has been updated to reflect changes in
```

File: omero/sysadmins/whatsnew.rst [hunk 7]
```diff
@@ -1,35 +1,23 @@
...
-  use-case in Dundee. See :ref:`hosting_data_example` for more info.
+  separately from or with OMERO.server on Ubuntu 18.04.
 
-- :doc:`restricted-admins` can now be created to allow
-  facility managers, image analysts etc. to organize users and data in OMERO
-  without having to be granted full administrator privileges. Full
-  administrators and administrators with restricted privileges can now create
-  Project/Dataset/Screen on behalf of other users in OMERO.web or transfer all
-  the data of a given user using the :program:`chown` command.
+- Add configuration option to fix OpenSSL issue.
```
Comment (mtbc) [inline omero/sysadmins/whatsnew.rst]:
```suggestion
- Add ``omero.glacier2.IceSSL.*`` configuration options to fix OpenSSL issue.
```

**Review by mtbc — APPROVED** [review]

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/64/ is green

**Pull request merged**